src/polly/src/epoll.rs

```diff
@@ -0,0 +1,201 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::io;
+use std::ops::Deref;
+use std::os::unix::io::{AsRawFd, RawFd};
+
+use libc::{
+    epoll_create1, epoll_ctl, epoll_event, epoll_wait, EPOLLERR, EPOLLET, EPOLLEXCLUSIVE, EPOLLHUP,
+    EPOLLIN, EPOLLONESHOT, EPOLLOUT, EPOLLPRI, EPOLLRDHUP, EPOLLWAKEUP, EPOLL_CLOEXEC,
+    EPOLL_CTL_ADD, EPOLL_CTL_DEL, EPOLL_CTL_MOD,
+};
+
+use utils::syscall::SyscallReturnCode;
+
+/// Refers to the operation to be performed on a file descriptor.
```

Comment:
nit: this comment needs a bit of re-wording. We can just say something like: 
```rust
\\\ Wrapper over EPOLL_CTL_* operations that can be performed on a file descriptor.
```

This way we also specify that we use this to wrap the control operations.